src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1269,6 +1278,7 @@ private void Filter_SelectionChanged(object sender, FilterChangedEventArgs e)
 
                 NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(async () =>
                 {
+                    await _packageDetail._packageDetailsTabControl.PackageDetailsTabViewModel.SetCurrentFilterAsync(_topPanel.Filter);
```

Comment:
I don't recall this change before but am curious about the design choice of passing the topPanel's tab to be stored and managed inside the PackageDetailsTabViewModel. 
It's already available in the PackageManagerControl which is at the top of the hierarchy that creates these TabViewModels. It can just be passed with `Control.ActiveFilter` or with some refactoring it could be part of one of the parent ViewModels.

Reply:
The current tab is used to determine if local readmes are being rendered. I switched it to use ActiveFilter, but I do think it's necessary for the ReadmePreviewViewModel to know what the current tab is so it can handle rendering. @jebriede  and I discussed how this would be better met with an event that the VM would listen to and can update itself but that would require a refactoring that feels outside of the scope of this PR.

Reply:
> I don't recall this change before but am curious about the design choice of passing the topPanel's tab to be stored and managed inside the PackageDetailsTabViewModel. It's already available in the PackageManagerControl which is at the top of the hierarchy that creates these TabViewModels. It can just be passed with `Control.ActiveFilter` or with some refactoring it could be part of one of the parent ViewModels.

I agree with this idea. I also think it's a good idea not to manage state of the top panel in a view model that pertains to something else. It's fine if that view model needs to react to a change in the top panel's tab, but storing it now requires it to manage the state of its internal representation of what the top panel's tab is and that's introducing the need to synchronize the state of that variable with whatever the top panel is.

Reply:
Thinking about this for now I want to leave the control of the visibility of the Readme tab inside of ReadmePreviewViewModel. 

I am changing the name of the function to ItemFilterChangedAsync which I think is more accurate to what we want. The VM doesn't need to store the current state it just needs to respond to when the filter changes. Since this view is the one that currently updates the filter on the DetailControlModel, I think it makes sense to let it also set the filter for the ReadmePreviewViewModel. Though I changed it so it fires the change directly on ReadmePreviewViewModel instead of a method inside of the PackageDetailsTabViewModel.